community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java

```diff
@@ -508,8 +508,20 @@
 
     public enum LabelIndex
     {
+        /**
+         * Native label index. Generally the best option.
+         */
         NATIVE,
-        LUCENE;
+
+        /**
+         * Label index backed by Lucene.
+         */
+        LUCENE,
+
+        /**
+         * Selects which ever label index is present in a store, or the default (NATIVE) if no label index present.
+         */
+        AUTO;
```

Comment:
Why not to use AUTO as default property value and then do fallback to NATIVE in case if none of the indexes exist?